### PR TITLE
fix(gv-autocomplete): close options panel when selecting option

### LINKED ISSUE #296: gv-autocomplete options panel should close when an item is selected
### Current behavior

In the case of a gv-autocomplete component with "no option" slot, when the user clicks on an item to select it, the item is selected but item list is not closed.

https://user-images.githubusercontent.com/1655950/109662489-68820280-7b6b-11eb-8e45-e81b23eda0d3.mp4

### Expected behavior

Item selection panel should be closed when an item is selected.

### PR DESCRIPTION
**Pull request by loriepisicchio**
Fixes #296

### CODE REVIEW

File: src/atoms/gv-autocomplete.js [hunk 1]
```diff
@@ -270,11 +274,17 @@ export class GvAutocomplete extends LitElement {
...
     };
+
+    const noOptionClasses = {
+      shouldShowNoOption,
+    };
     return html` <div class="container">
       ${this._renderStyle()}
       <slot></slot>
-      <div class="${classMap(classes)}" @mouseleave="${this._onMouseLeave}" style="${styleMap({ top: `${top}px` })}">
+      <div class="${classMap(noOptionClasses)}" style="${styleMap({ top: `${top}px` })}">
```
Comment (gcusnieux) [inline src/atoms/gv-autocomplete.js:284]: Where is mouseleave binding ?

**Review by gaetanmaisse — CHANGES_REQUESTED** [review]
I think there is something strange about `shouldShowNoOption` as it doesn't look to be really used 🤔  I will ping you on slack

File: src/atoms/gv-autocomplete.js (hunk 1 above)
Comment (loriepisicchio) [reply src/atoms/gv-autocomplete.js:284]: It is at L286, on the options div. This is a new div that contains only `noOption` slot.

**Comment by gaetanmaisse** [comment]
I added a new commit as it looked like `shouldShowNoOption` wasn't necessary

**Review by gaetanmaisse — APPROVED** [review]

**Review by gcusnieux — APPROVED** [review]

**Pull request merged**